packages/components/src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Element } from '../Element';
+import { IconButton } from '../IconButton';
+
+interface BannerProps {
+  children: React.ReactNode;
+  onDismiss?: () => void;
+}
+
+export const Banner = ({ children, onDismiss }: BannerProps) => {
+  return (
+    <Element
+      css={{
+        position: 'relative',
+        padding: [4, 6, 8],
```

Comment:
is this a styled-component or a styled-system feature? I thought it only works with the css() function

Reply:
Styled System! It does only work with the `css()` function, yes. The `Element` component uses it to generate the component, so we can pass it with the `css` prop:
```tsx
export const Element = styled.div<IElementProps>(props =>
  css({
    // stuff
    ...(props.css || {}),
  })
);
  ```